new-site/docs-site/content/0.19.0/guide/typesense-dynamodb.md

```diff
@@ -0,0 +1,240 @@
+# DynamoDB Integration with Typesense
+
+Hey there! This post give you a guide to integrate Typesense cluster with AWS DynamoDB by setting up a trigger with AWS Lambda.
+
+![Typesense DynamoDB Integration Chart](~@guide-images/typesense-dynamodb/typesense-dynamodb.svg)
+
+## Step 1: Create Typesense Cluster
+
+Sign up for a new account in [Typesense Cloud](https://cloud.typesense.org/) and get Endpoint URL, Port number and API key.
+## Step 2: Create a DynamoDB table
+
+Create a DynamoDB table with your choice of name and partition key ("id" is recommended).
+
+After creating a DynamoDB table you want to enable streams in the "Overview" section.
+
+You can also do this using AWS CLI:
+
+```bash
+aws dynamodb create-table \
+    --table-name typensense \
+    --attribute-definitions AttributeName=id,AttributeType=N \
+    --key-schema AttributeName=id,KeyType=HASH  \
+    --provisioned-throughput ReadCapacityUnits=5,WriteCapacityUnits=5 \
```

Comment:
Is `--provisioned-throughput ReadCapacityUnits=5,WriteCapacityUnits=5` a required parameter? 

Could we only have the absolute required parameters, in order to make it as simple as possible?

Reply:
If we use that one we can stay on free tier for a long period this is the minimum configuration. But it is not necessary.